pkg/client/client.go

```diff
@@ -153,7 +153,7 @@ func newKubeClient(config *restclient.Config) (controllercli.Client, error) {
 	return controlCli, nil
 }
 
-func buildKubeConnectionConfig() (*restclient.Config, error) {
+func BuildKubeConnectionConfig() (*restclient.Config, error) {
```

Comment:
@ricardozanini To access the pod logs I use client from "k8s.io/client-go/kubernetes", do you know about any simpler way?
Also to use the client I need access to kubeconfig (that is why I exported this function). Should I move it to infrastructure?

Reply:
@sutaakar if you need a specific method from the Pod API, please exposed it instead of the `buildKube..` function. On our code base, we aren't accessing the kube client directly, please see this implementation as an example:
https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/client/openshift/deploymentconfig.go

In this case, you create a `pod.go` file within the `kubernetes` package. :)

Reply:
Please take another look, implementation refactored and moved to `pod.go`